.pipelines/templates/acr-cleanup.yaml

```diff
@@ -1,8 +1,12 @@
 steps:
   - bash: |
-      az login --identity
-      echo ${CHECKOUT_TAG}
-      az acr repository delete -n $(registry.name) --image oss/openservicemesh/osm-chart:${CHECKOUT_TAG} --yes
+      if [[ -n "${CHECKOUT_TAG}" ]]; then
```

Comment:
are we also checking `CHECKOUT_TAG` for publish-helm-chart?

Reply:
Good catch, added it to publish-helm-chart.yaml